src/components/Admin/AdminLog.vue

```diff
@@ -1,102 +1,78 @@
 <template>
-  <v-container
-    fluid
+  <v-card
+    max-width="1075"
+    class="mx-auto"
   >
-    <v-data-table
-      v-model="selected"
-      :headers="headers"
-      :items="devices"
-      :single-select="singleSelect"
-      item-key="reservationID"
-      show-select
-      class="elevation-1"
+    <v-toolbar
+      color="light-grey"
+      light
     >
-      <template v-slot:top>
-        <v-switch
-          v-model="singleSelect"
-          label="Single select"
-          class="pa-3"
-        />
-      </template>
-      <template v-slot:[`item.status`]="{ item }">
-        <v-chip
-          :color="getColor(item.status)"
-          text-color="black"
-        >
-          {{ item.status }}
-        </v-chip>
-      </template>
-    </v-data-table>
-  </v-container>
+      <v-toolbar-title>Reservations Log</v-toolbar-title>
+
+      <v-spacer />
+    </v-toolbar>
+
+    <v-list
+      subheader
+    >
+      <v-list-item
+        v-for="device in devices"
+        :key="device.name"
+      >
+        <v-list-item-avatar>
+          <v-icon
+            :color="getColor(device.status)"
+            light
+          >
+            bookmark
+          </v-icon>
+        </v-list-item-avatar>
+
+        <v-list-item-content>
+          <v-list-item-title v-text="device.deviceName" />
+
+          <v-list-item-subtitle v-text="'Reservation ID: ' + device.reservationID" />
+
+          <v-list-item-subtitle v-text="'Device Tag: ' + device.deviceTag" />
+
+          <v-list-item-subtitle v-text="'Username: ' + device.username" />
+
+          <v-list-item-subtitle v-text="'First Name: ' + device.firstName" />
+
+          <v-list-item-subtitle v-text="'Last Name: ' + device.lastName" />
+
```

Comment:
lots of unnecessary spacing
```suggestion
          <v-list-item-subtitle v-text="'Reservation ID: ' + device.reservationID" />
          <v-list-item-subtitle v-text="'Device Tag: ' + device.deviceTag" />
          <v-list-item-subtitle v-text="'Username: ' + device.username" />
          <v-list-item-subtitle v-text="'First Name: ' + device.firstName" />
          <v-list-item-subtitle v-text="'Last Name: ' + device.lastName" />
```